src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -467,6 +467,7 @@ def get(self):
 
 
 @UiApiController('/cephfs', Scope.CEPHFS)
+@ControllerDoc("Ui helper function; not part of the public API", "CephFSUi")
```

Comment:
Small nit: how about changing this to:
```suggestion
@ControllerDoc("Dashboard UI helper function; not part of the public API", "CephFSUi")
```

Reply:
Sounds good!